src/components/tests/ai-input.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import React from 'react';
+import { AIInput } from '../ai-input';
+
+describe( 'AIInput Component', () => {
+	let handleSend: jest.Mock;
+	let handleKeyDown: jest.Mock;
+	let setInput: jest.Mock;
+	let clearInput: jest.Mock;
+
+	const defaultProps = {
+		disabled: false,
+		input: '',
+		setInput: jest.fn(),
+		handleSend: jest.fn(),
+		handleKeyDown: jest.fn(),
+		clearInput: jest.fn(),
+		isAssistantThinking: false,
+	};
+
+	const getInput = () => screen.getByTestId( 'ai-input-textarea' ) as HTMLTextAreaElement;
```

Comment:
Are we able to use a user-facing query instead? E.g., the placeholder or ARIA label. Doing so is a more stable and user-oriented approach IMO.